bootstrap/src/main/config/mediaserver.xml

```diff
@@ -80,6 +80,6 @@
 			TLS_RSA_WITH_AES_256_CBC_SHA256, TLS_RSA_WITH_AES_128_CBC_SHA256,
 			TLS_RSA_WITH_AES_256_CBC_SHA, TLS_RSA_WITH_AES_128_CBC_SHA,
 			TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256</cipherSuites>
-		<certificate path="x509-server.pem" key="x509-server-key.pem" algorithm="rsa"/>
+		<certificate path="../conf/dtls/x509-server.pem" key="../conf/dtls/x509-server-key.pem" algorithm="rsa"/>
```

Comment:
Make the ECDSA the default certificate. It is more secure and much faster. RSA will be deprecated in Chrome in the near future as well.